CONTRIBUTING.md

```diff
@@ -86,6 +87,51 @@ assessment.
 
 ## Discussing Mozilla's Position on a Web Specification
 
+### For Mozilla's subject-matter experts
+
+You might be asked for your opinion
+on specifications where you are one of the experts within the Mozilla community,
+or feel like you have valuable contributions to make
+to issues on other specifications even where you haven't explicitly been asked to do so.
+This is the purpose of this repository:
```

Comment:
Maybe move this to be the first sentence and rephrase? Something like:
 
"The purpose of this repository is to help Mozilla's technical leaders come to a position about new features being added to the Web. As such, you might be asked for your opinion..."

Reply:
I did a variant of what you suggested.